core/services/keeper/registry_synchronizer_log_listener.go

```diff
@@ -33,31 +34,46 @@ func (rs *RegistrySynchronizer) HandleLog(broadcast log.Broadcast) {
 	case *registry1_1.KeeperRegistryKeepersUpdated,
 		*registry1_1.KeeperRegistryConfigSet,
 		*registry1_2.KeeperRegistryKeepersUpdated,
-		*registry1_2.KeeperRegistryConfigSet:
+		*registry1_2.KeeperRegistryConfigSet,
+		*registry1_3.KeeperRegistryDevKeepersUpdated,
+		*registry1_3.KeeperRegistryDevConfigSet:
 		// same mailbox because same action for config set and keepers updated
 		wasOverCapacity = rs.mailRoom.mbSyncRegistry.Deliver(broadcast)
 		mailboxName = "mbSyncRegistry"
 	case *registry1_1.KeeperRegistryUpkeepCanceled,
-		*registry1_2.KeeperRegistryUpkeepCanceled:
+		*registry1_2.KeeperRegistryUpkeepCanceled,
+		*registry1_3.KeeperRegistryDevUpkeepCanceled:
 		wasOverCapacity = rs.mailRoom.mbUpkeepCanceled.Deliver(broadcast)
 		mailboxName = "mbUpkeepCanceled"
 	case *registry1_1.KeeperRegistryUpkeepRegistered,
-		*registry1_2.KeeperRegistryUpkeepRegistered:
+		*registry1_2.KeeperRegistryUpkeepRegistered,
+		*registry1_3.KeeperRegistryDevUpkeepRegistered:
 		wasOverCapacity = rs.mailRoom.mbUpkeepRegistered.Deliver(broadcast)
 		mailboxName = "mbUpkeepRegistered"
 	case *registry1_1.KeeperRegistryUpkeepPerformed,
-		*registry1_2.KeeperRegistryUpkeepPerformed:
+		*registry1_2.KeeperRegistryUpkeepPerformed,
+		*registry1_3.KeeperRegistryDevUpkeepPerformed:
 		wasOverCapacity = rs.mailRoom.mbUpkeepPerformed.Deliver(broadcast)
 		mailboxName = "mbUpkeepPerformed"
-	case *registry1_2.KeeperRegistryUpkeepGasLimitSet:
+	case *registry1_2.KeeperRegistryUpkeepGasLimitSet,
+		*registry1_3.KeeperRegistryDevUpkeepGasLimitSet:
 		wasOverCapacity = rs.mailRoom.mbUpkeepGasLimitSet.Deliver(broadcast)
 		mailboxName = "mbUpkeepGasLimitSet"
-	case *registry1_2.KeeperRegistryUpkeepReceived:
+	case *registry1_2.KeeperRegistryUpkeepReceived,
+		*registry1_3.KeeperRegistryDevUpkeepReceived:
 		wasOverCapacity = rs.mailRoom.mbUpkeepReceived.Deliver(broadcast)
 		mailboxName = "mbUpkeepReceived"
-	case *registry1_2.KeeperRegistryUpkeepMigrated:
+	case *registry1_2.KeeperRegistryUpkeepMigrated,
+		*registry1_3.KeeperRegistryDevUpkeepMigrated:
 		wasOverCapacity = rs.mailRoom.mbUpkeepMigrated.Deliver(broadcast)
 		mailboxName = "mbUpkeepMigrated"
+	case *registry1_3.KeeperRegistryDevUpkeepPaused:
+		svcLogger.Debug("delivered to upkeep paused")
```

Comment:
nit. I like having more debug logs, could you add them to all the different types :D

Reply:
Done